### ISSUE TITLE
PolSIR BT

### ISSUE THREAD

**Issue by chengdang** [JCSDA/CRTMv3#187]
@BenjaminTJohnson 
Here are some hofx created using the PolSIR coefficients:

![PolSIR BT Chan1](https://github.com/user-attachments/assets/bde0a5f4-9eb3-4164-af7e-c9fcebd85080)
![PolSIR BT Chan2](https://github.com/user-attachments/assets/e4e6d153-d720-4cdc-9533-a94e2719a249)
![PolSIR BT Chan3](https://github.com/user-attachments/assets/3c3a91c6-dbcb-490f-8be5-d308fe9ee943)
![PolSIR BT Chan4](https://github.com/user-attachments/assets/4c21255b-6f08-4e3d-ade5-70d06a848053)
![PolSIR BT Chan5](https://github.com/user-attachments/assets/ae076ed0-a496-4f4c-9e75-e5053eed9f0d)
 litt
![PolSIR BT Chan6](https://github.com/user-attachments/assets/7fc2f49b-a0fe-4a5e-b8a4-f06dbc3e53f4)

**Comment by chengdang** [comment]
The above BT were computed with an altered ctest: `ufo_test_tier1_test_ufo_opr_atms_crtm.`
In which the `atms` spectral and tau coefficients are replaced with the `polsir` coefficients (little endian).

Here is the yaml file:

```
time window:
  begin: 2018-04-14T21:00:00Z
  end: 2018-04-15T03:00:00Z

observations:
- obs operator:
    name: CRTM
    Absorbers: [H2O,O3,CO2]
    Clouds: [Water, Ice]
    Cloud_Fraction: 1.0
    SurfaceWindGeoVars: uv
    linear obs operator:
      Absorbers: [H2O,O3,CO2]
      Clouds: [Water, Ice]
    obs options:
      Sensor_ID: atms_npp
      EndianType: little_endian
      CoefficientPath: Data/
      NC_CoefficientPath: Data/
      CloudCoeff_File: CloudCoeff.nc4
      CloudCoeff_Format: netCDF
      Cloud_Model: CRTM
      AerosolCoeff_File: AerosolCoeff.nc4
      AerosolCoeff_Format: netCDF
      Aerosol_Model: CRTM
  obs space:
    name: Radiance
    obsdatain:
      engine:
        type: H5File
        obsfile: Data/ufo/testinput_tier_1/atms_npp_obs_2018041500_m_qc.nc4
    obsdataout:
      engine:
        type: H5File
        obsfile: Data/polsir_obs_2018041500_m_qc_polsir_crtm_out.nc4
    simulated variables: [brightnessTemperature]
    channels: 1-6
  geovals:
    filename: Data/ufo/testinput_tier_1/atms_npp_geoval_2018041500_m_qc.nc4
  vector ref: GsiHofX
  tolerance: 1.e-7
  linear obs operator test:
    iterations TL: 5
    tolerance TL: 2.0e-5
    tolerance AD: 1.0e-11

```

**Comment by BenjaminTJohnson** [comment]
Thanks!

the colorbars don't seem to be showing any numbers for me -- it might be that I'm using dark mode, and the numbers are black on a transparent background (yes, that's it -- my problem, not yours).  

Is it relatively easy to create a histogram of TBs per channel?

**Comment by chengdang** [comment]
Yes it is. Let me update soon.

**Comment by chengdang** [comment]
Here are the histogram plot:

![PolSIR BT Hist](https://github.com/user-attachments/assets/5e4a1179-49e4-4919-984c-ca7ed8586e33)

**Comment by BenjaminTJohnson** [comment]
Hmm.  Visually channels 3 and 4 look identical, and channels 5 and 6 look identical, and they should be different even though the frequency is the same, the polarization should be different -- so over ocean points should be different.

**Comment by chengdang** [comment]
Yes, you are right, hofx of these paired channels are identical. Hmmm

**Comment by BenjaminTJohnson** [comment]
I can modify the polarization, would you be able to easily re-run the experiment using a new coefficient?

**Comment by chengdang** [comment]
Yes. I can do that easily. Let me know when it's ready.

**Comment by BenjaminTJohnson** [comment]
@chengdang 
https://bin.ssec.wisc.edu/pub/s4/CRTM/file/polsir_p01_9_10.SpcCoeff.nc

**Comment by chengdang** [comment]
@BenjaminTJohnson  could you convert this into LE binary file? The UFO pipeline is not ready for netCDF versions of spc and tau coefficient yet (no TauCoeff_Format and SpcCoeff_Format yet).

**Comment by BenjaminTJohnson** [comment]
@chengdang https://bin.ssec.wisc.edu/pub/s4/CRTM/file/polsir_p01_9_10.SpcCoeff.bin  you can use the existing polsir taucoeff file.

**Comment by chengdang** [comment]
@BenjaminTJohnson Thank you! There are some minor difference between these paired channels, but still almost identical:
![PolSIR BT Scatter](https://github.com/user-attachments/assets/10244bb6-dbb5-41d7-ac3c-fc301026a117)

**Comment by BenjaminTJohnson** [comment]
Thanks -- it's entirely possible that the surface is obscured by water vapor, and we won't see polarization effects from the surface.  The other big thing that's missing is O3 contribution.
https://amt.copernicus.org/articles/13/53/2020/ (fig 1)

We could determine whether surface is obscured by plotting weighting functions / jacobians wrt the surface.
